tickets/Ticket.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp8
+{
+    public class Ticket
+    {
+        public Guid ID { get; set; }
+        public DateTime DateCreated { get; set; }
+
+        public Ticket()
+        {
+            ID = Guid.NewGuid();
+            DateCreated = DateTime.Now;

```

Comment:
Use `DateTime.UtcNow` to avoid issues with timezones. Its best practice at majority of commercial projects